pubspec.yaml

```
@@ -45,6 +46,9 @@ dependencies:
   permission_handler: ^3.3.0
   provider: ^4.0.0
   rxdart: ^0.22.3
+  share:
+    git:
+      url: https://github.com/d-silveira/flutter-share.git
```

Even though this package seems to be dead, I'd prefer a fixed version instead of a potentially unstable master.
```suggestion
  share:
    git:
      url: https://github.com/d-silveira/flutter-share.git
      ref: 1f8b139
```